add first draft of jax diffusers community event. (#133)
huggingface/community-events: jax-controlnet-sprint/RAEDME.md

# JAX/Diffusers community week 🧨

> Welcome to the JAX/Diffusers community week! The goal of this week is to work on fun and creative Diffusion models using JAX and Diffusers.

Reviewer: ```suggestion
Welcome to the JAX/Diffusers community sprint! The goal of this sprint is to work on fun and creative diffusion models using JAX and Diffusers.
```

> Welcome to the FJAX/Diffusers community week! The goal of this week is to work on fun and creative Diffusion models using JAX and Diffusers.

Reviewer 2: ```suggestion
Welcome to the JAX/Diffusers community week! The goal of this week is to work on fun and creative Diffusion models using JAX and Diffusers.
```

> To do so, we will try to teach **you** how to effectively use JAX/Flax on TPU and help you to complete a fun project that involves Diffusion models in JAX/Flax during the community week. 

Reviewer: ```suggestion
In this event, we will do various applications with diffusion models in JAX/Flax and diffusers using free TPU hours provided by Google Cloud.
```
Try to keep the language to _we_

Free access to a TPU v4-8 will kindly be provided by the Google Cloud team!

> In this document, we list all the important information that you will need during the JAX/Diffusers community week week.

Reviewer 2: ```suggestion
In this document, we list all the important information that you will need during the JAX/Diffusers community week.
```

Don't forget to sign up [here] (TODO)! 

> > 💡 Note: This document is still WIP and it only contains initial details of the event. We will keep updating this document as we make other relevant information available throughout the community week. 

Reviewer: ```suggestion
> 💡 Note: This document is still WIP and it only contains initial details of the event. We will keep updating this document as we make other relevant information available throughout the community sprint. 
```

## Table of Contents

- [Organization](#organization)
- [Important dates](#important-dates)
- [Communication](#communication)

## Organization 

Participants can propose ideas for an interesting project involving Diffusion models. Teams of 3 to 5 will then be formed around the most promising and interesting projects. Make sure to read through the [Projects] (TODO) section on how to propose projects, comment on other participants' project ideas, and create a team.

> To help each team successfully finish their project, we will organize talks by leading scientists and engineers from Google, Hugging Face, and the open-source Diffusion community. The talks will take place before the community week from March 29th to May 1st. Make sure to attend the talks to get the most out of your participation! Check out the [Talks] (TODO) section to get an overview of the talks, including the speaker and the time of the talk.

Reviewer: ```suggestion
To help each team successfully finish their project, we will organize talks by leading scientists and engineers from Google, Hugging Face, and the open-source diffusion community. The talks will take on 17th of April. Make sure to attend the talks to get the most out of your participation! Check out the [Talks] (TODO) section to get an overview of the talks, including the speaker and the time of the talk.
```

> Each team is then given **free access to a TPU v4-8 VM** from April 14 to May 1st. In addition, we will provide a training example in JAX/Flax and Diffusers to train [ControlNets](https://huggingface.co/blog/controlnet) to kick-start your project. We will also provide examples of how to prepare datasets for ControlNet training. During the week, we'll make sure to answer any questions you might have about JAX/Flax and Diffusers and help each team as much as possible to complete their project!

Reviewer: ```suggestion
Each team is then given **free access to a TPU v4-8 VM** from April 14 to May 1st. In addition, we will provide a training example in JAX/Flax and Diffusers to train [ControlNets](https://huggingface.co/blog/controlnet) to kick-start your project. We will also provide examples of how to prepare datasets for ControlNet training. During the sprint, we'll make sure to answer any questions you might have about JAX/Flax and Diffusers and help each team as much as possible to complete their projects!
```

> Each team is then given **free access to a TPU v4-8 VM** from April 14 7th to May 1st. In addition, we will provide a training example in JAX/Flax and Diffusers to train [ControlNets](https://huggingface.co/blog/controlnet) to kick-start your project. Additionally, we will also provide examples of how to prepare datasets for ControlNet training. During the week, we'll make sure to answer any questions you might have about JAX/Flax and Diffusers and help each team as much as possible to complete their project!

Reviewer 2: ```suggestion
Each team is then given **free access to a TPU v4-8 VM** from April 14 to May 1st. In addition, we will provide a training example in JAX/Flax and Diffusers to train [ControlNets](https://huggingface.co/blog/controlnet) to kick-start your project. We will also provide examples of how to prepare datasets for ControlNet training. During the week, we'll make sure to answer any questions you might have about JAX/Flax and Diffusers and help each team as much as possible to complete their project!
```

> At the end of the community week, each team should submit a demo of their project. All demonstrations will be evaluated by a jury and the top-3 demos will be awarded a prize. Check out the [How to submit a demo] (TODO) section for more information and suggestions on how to submit your project.

Reviewer: ```suggestion
At the end of the community sprint, each submission will be evaluated evaluated by a jury and the top-3 demos will be awarded a prize. Check out the [How to submit a demo] (TODO) section for more information and suggestions on how to submit your project.
```

> 💡 Note: Even though we provide an example for performing ControlNet training, participants can propose ideas that do not involve ControlNets at all. But the ideas need to be centered around Diffusion models.

> ## Important dates

Reviewer 2: When are the talks?

- **29.03.** Official announcement of the community week. Make sure to sign-up in [this google form] (TODO).
- **03.04.** Start forming groups.
- **10.04.** Officially start the week. This phase will mostly focus on data collection, talks, etc. 
- **14.04. - 17.04.** Start providing access to TPUs. 
- **01.05.** Shutdown access to TPUs. 
- **08.05.**: Project presentation.

## Communication

> All important communication will take place on our Discord server (TODO: update with more information).

Reviewer 2: You can add join link in hf.co/join/discord and mention that they will be added to some channels, but they need to sign the form first

> Important announcements of the Hugging Face, Flax/JAX, and Google Cloud team will be posted there. 

Reviewer: ```suggestion
Important announcements of the Hugging Face, Flax/JAX, and Google Cloud team will be posted in the server.
```

> Such announcements include general information about the community week (Dates, Rules, ...), release of relevant training scripts (Flax/JAX example scripts for NLP and Vision), release of other important documents (How to access the TPU), etc. 

Reviewer: ```suggestion
```
Most of them will be included in this document so it'd be better to remove this IMO. We try to keep announcements less to spam folks less.

> The Discord channel will also be the central place for participants to post about their results, share their learning experiences, ask questions, etc.

Reviewer: ```suggestion
The discord server will be the central place for participants to post about their results, share their learning experiences, ask questions and get technical support in various obstacles they encounter.
```

> For issues with Flax/JAX, Diffusers, Datasets or for questions that are specific to your project we would be **very happy** if you could use the following public repositories and forums:

Reviewer: ```suggestion
For issues with Flax/JAX, Diffusers, Datasets or for questions that are specific to your project we will be interacting through public repositories and forums:
```

- Flax: [Issues](https://github.com/google/flax/issues), [Questions](https://github.com/google/flax/discussions)
- JAX: [Issues](https://github.com/google/jax/issues), [Questions](https://github.com/google/jax/discussions)
- 🤗 Diffusers: [Issues](https://github.com/huggingface/diffusers/issues), [Questions](https://discuss.huggingface.co/c/discussion-related-to-httpsgithubcomhuggingfacediffusers/63)
- 🤗 Datasets: [Issues](https://github.com/huggingface/datasets/issues), [Questions](https://discuss.huggingface.co/c/datasets/10)
- Project specific questions: [TODO]()
- TPU related questions: [TODO]()

> Please do **not** post the complete issue/project-specific question in the Slack channel, but instead a link to your issue/question that we will try to answer as soon as possible. 

Reviewer 2: ```suggestion
Please do **not** post the complete issue/project-specific question in Discord, but instead a link to your issue/question that we will try to answer as soon as possible. 
```

> Please do **not** post the complete issue/project-specific question in Discord, but instead a link to your issue/question that we will try to answer as soon as possible. 

Reviewer: ```suggestion
It's appreciated to post the link to forum post to the discord server, instead of directly posting issues or questions.
```

> This way, we make sure that the everybody in the community can benefit from your questions - even after the community week - and that the same question is not answered twice.

Reviewer: ```suggestion
This way, we make sure that the everybody in the community can benefit from your questions, even after the community sprint.
```

> To be invited to the Discord server, please make sure you have signed up [on the Google form] (TODO). 

Reviewer: ```suggestion
```
This is not the case as of now

Author: How are we handling it?
